src/ui/IngredientForm.tsx

```diff
@@ -0,0 +1,24 @@
+import { FieldArray } from 'formik';
```

Comment:
Let's just call this file, Ingredients. The current name makes it sound like you're creating a new <form />